### PR TITLE
Improve how Jax array are handled (close #1136)

### LINKED ISSUE #1136: Improve how Jax array is handled
```py
import jax.numpy as jnp
import lets_plot as lp
import numpy

(
    lp.ggplot(
        {"x": list(jnp.arange(0, 10)), "y": numpy.arange(0, 10)},
        lp.aes(x="x", y="y"),
    )


    + lp.geom_line()
).show()
```
test
y axis is the expected behavior with numpy
x axis is the behavior when used with jax
(jax: https://jax.readthedocs.io/en/latest/index.html)

### PR DESCRIPTION
**Pull request by sunnyayyl**
See #1136, this makes jax array treated and handled the same way numpy ndarray get treated 

(Apologies in advance for any rookie mistake I make, first time making a pull request and first time contributing to anything in general)

### CODE REVIEW

**Comment by sunnyayyl** [comment]
As a comparison
### Before
352269585-2262d6fc-0ce7-44e0-bd31-b52a159c9dc2
### After
354931469-7fdbbfbd-6eb2-4f02-a33b-d1d97e3a101e

**Comment by alshan** [comment]
Cool, thanks @sunnyayyl !
